library/common/main_interface.h

```diff
@@ -83,6 +83,14 @@ envoy_status_t reset_stream(envoy_stream_t stream);
  */
 envoy_engine_t init_engine();
 
+/**
+ * Update the network interface to the preferred network for opening new connections.
```

Comment:
Is this correct? Shouldn't we be maintaining two _connections_ and switching between them when this function is called then piping future _requests_ over this connection?

Reply:
Yes, I think new _streams_ is actually more accurate. Will update comment.

Reply:
Yep, I like @rebello95's suggestion better than what we had previously.